packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -439,13 +439,13 @@ async function transformGraphQLSchema(context, options) {
   }
 
   const buildConfig = {
+    ...options,
     buildParameters,
     projectDirectory: options.dryrun ? false : resourceDir,
     transformersFactory: transformerListFactory,
     transformersFactoryArgs: [searchableTransformerFlag, storageConfig],
     rootStackFileName: 'cloudformation-template.json',
     currentCloudBackendDirectory: previouslyDeployedBackendDir,
-    disableResolverOverrides: options.disableResolverOverrides,
```

Comment:
Why are we removing disableResolverOverrides here?

Reply:
Because we are spreading the value from options and this option will be set only when using mock